.github/workflows/deploy-storybook.yml

```
@@ -0,0 +1,43 @@
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      # 1️⃣ 레포 체크아웃
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      # 2️⃣ Node.js 설정
+      - name: Setup Node
+        uses: actions/setup-node@v3
```

actionlint (1.7.8)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)